src/networks.js

```diff
@@ -188,8 +196,6 @@ export function getAgreementConnectorConfig(chainId) {
   }
 }
 
-export const SUPPORTED_CHAINS = [4, 100, 137] // Add  arbitrum  chains id + fill the network json with the data
-
 export function isSupportedChain(chainId) {
```

Comment:
It's possible translate that file to TS?

Reply:
I can try 👌

Reply:
I tried but it has too many ramifications.